R/attr.R

```diff
@@ -110,11 +110,16 @@ detect_void_name <- function(x) {
 is_dictionaryish <- function(x) {
   # 2022-01: Used in many packages. Don't deprecate without a
   # replacement.
-  if (!length(x)) {
-    return(!is.null(x))
-  }
+  if (is.null(x)) {
+    TRUE
+  } else {
+    if (!length(x)) {
+      return(!is.null(x))
+    }
 
-  is_named(x) && !any(duplicated(names(x)))
+    is_named(x) && !any(duplicated(names(x)))
+
```

Comment:
Can you remove this new unnecessary whitespace please?

Reply:
done!